cap/modules/experiments/validators.py

```diff
@@ -59,3 +60,12 @@ def validate_cadi_id(validator, value, instance, schema):
     from .utils.cadi import get_from_cadi_by_id
     if not get_from_cadi_by_id(instance, from_validator=True):
         yield ValidationError("{} not found in CADI.".format(instance))
+
+
+def validate_unique_cadi(validator, value, instance, schema):
+    from .utils.cadi import get_deposit_by_cadi_id
+    try:
+        get_deposit_by_cadi_id(instance)
```

Comment:
might be better if we return a validation error with a link to the draft of the same CADI id